src/common/tools/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "no-console": "off",
+    "no-process-exit": "off",
+    "node/no-unpublished-require": "off",
```

Comment:
Maybe someday we should actually move these (actually, most) devDependencies into dependencies. It seems silly but there's a real possibility a project might add the cts as a dependency (e.g. webgpu for node). We would need to test it though because I have no idea what will happen if you actually try to do this.